git_machete/client.py

```diff
@@ -570,6 +588,9 @@ def slide_out(
                     new_downstream,
                     opt_no_interactive_rebase)
 
```

Comment:
Hmm if this method is going to be called from `clean`... should the lines 575-589 be executed? 🤔

Reply:
Why not? if the branch about to be deleted has downstream branch then merge/rebase is needed, no?

Reply:
Hmmm okay right, let's leave indeed